src/main/java/org/datadog/jenkins/plugins/datadog/apm/TracerLanguage.java

```diff
@@ -2,6 +2,7 @@
 
 public enum TracerLanguage {
     DOTNET(".NET"),
+    GO("Golang"),
```

Comment:
```suggestion
    GO("Go"),
```
Should we use the official name `Go` here? Or it's a requirement that it should be name `Golang`?

Reply:
Makes sense, renamed